server/crawler/crawlerController.js

```diff
@@ -21,9 +21,11 @@ const crawlerController = {
   
   // Sets up a scrape to run on an interval
   // Currently scrapes trulia only
-  startScrapeInterval: async (endpoint, interval, url) => {
+  startScrapeInterval: async (req, res) => {
```

Comment:
We should look into the consequences of using async on Express middleware. I'm afraid there isn't good rejection handling as it currently is built.